### PR TITLE
Raise the upper limit of MAX_JSON_SIZE

### PR DESCRIPTION
**Pull request by swfz**
about json cell in table visualization.

There was a part which could not be displayed partially, so it was set as the upper limit was checked

I want you to raise the upper limit of the number of character strings 

 Sorry, I'm not goog at English.

### CODE REVIEW

**Comment by swfz** [comment]
@arikfr 
Thank you for your feedback.
I fixed it so please check it.

**Comment by swfz** [comment]
@arikfr 
thank you for your comment!
I fixed it, so please check it.

File: redash/settings/__init__.py [hunk 1]
```diff
@@ -226,6 +226,7 @@ def all_settings():
 QUERY_REFRESH_INTERVALS = map(int, array_from_string(os.environ.get("REDASH_QUERY_REFRESH_INTERVALS", "60, 300, 600, 900, 1800, 3600, 7200, 10800, 14400, 18000, 21600, 25200, 28800, 32400, 36000, 39600, 43200, 86400, 604800, 1209600, 2592000")))
 PAGE_SIZE = int(os.environ.get('REDASH_PAGE_SIZE', 20))
 PAGE_SIZE_OPTIONS = map(int, array_from_string(os.environ.get("REDASH_PAGE_SIZE_OPTIONS", "5,10,20,50,100")))
+TABLE_CELL_MAX_JSON_SIZE = 200000
```
Comment (kravets-levko) [inline redash/settings/__init__.py]: Let's keep previous value `50000` as default (you can override it in your setup)

Comment (gabrieldutra) [reply redash/settings/__init__.py]: Does it make sense to get it as an environment var? :thinking: 
WDYT @kravets-levko, @arikfr ?

Comment (kravets-levko) [reply redash/settings/__init__.py]: @gabrieldutra Good point! Yes, let's take it from env with default `50000`

**Comment by swfz** [comment]
@kravets-levko 
I fixed it, so please check it.

**Comment by kravets-levko** [comment]
@arikfr, @gabrieldutra - do you have any notices or suggestions?

**Pull request merged**